package.json

```diff
@@ -1,17 +1,6 @@
 {
   "author": "Mike D Pilsbury <mike.pilsbury@gmail.com>",
-  "contributors": [
-    "Alex Robson",
-    "Arthur Schreiber",
-    "Bret Copeland <bret@atlantisflight.org> (https://github.com/bretcope)",
-    "Bryan Ross <bryan@rossipedia.com> (https://github.com/rossipedia)",
-    "Ciaran Jessup <ciaranj@gmail.com>",
-    "Cort Fritz <cfritz@caa.com>",
-    "lastonesky",
-    "Patrik Simek <patrik@patriksimek.cz>",
-    "Phil Dodderidge <pdodde@poyntz.com>",
-    "Zach Aller"
-  ],
+  "contributors": "https://github.com/tediousjs/tedious/graphs/contributors",
```

Comment:
I don't `contributors` accepts a single string to be set. 🤔

Instead of changing this here to a URL, I think we could automatically generate a list of authors from the git commit log before we publish the package via `semantic-release`.

Reply:
You are right 😅 Do you suggest generating a CONTRIBUTORS.md with list of authors, something with

```bash
$ git log --format='%aN' | sort -u > CONTRIBUTORS.md
```